microshift_configuring/microshift_low_latency/microshift-low-latency.adoc

```diff
@@ -0,0 +1,93 @@
+
+include::modules/microshift-low-latency-concept.adoc[leveloffset=+1]
+
+//additional resources for the low latency concept
+[role="_additional-resources"]
+.Additional resources
+* link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/scalability_and_performance/low-latency-tuning#cnf-understanding-low-latency_cnf-understanding-low-latency[About low latency] (OpenShift Container Platform documentation)
+
+include::modules/microshift-install-rpms-low-latency.adoc[leveloffset=+1]
+
+include::modules/microshift-low-latency-config-yaml.adoc[leveloffset=+1]
+
+//additional resources for the config.yaml
+[role="_additional-resources"]
+.Additional resources
+//TODO * workload partitioning crossref here
+* xref:../../microshift_configuring/microshift-using-config-tools.adoc#microshift-config-yaml_microshift-configuring[Using a YAML configuration file]
+* link:https://kubernetes.io/docs/reference/config-api/kubelet-config.v1beta1/#kubelet-config-k8s-io-v1beta1-KubeletConfiguration[KubeletConfiguration reference] (Kubernetes upstream documentation)
+
+//RHEL TuneD
+include::modules/microshift-low-latency-tuned-conc.adoc[leveloffset=+1]
+
+//microshift-baseline is the name of the profile and used for automatic activation settings
+//microshift-baseline-variables.conf is the file for user tweaks
+include::modules/microshift-low-latency-tuned-profile.adoc[leveloffset=+2]
+
+//additional resources for tuned profiles
+[role="_additional-resources"]
+.Additional resources
+* link:https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/monitoring_and_managing_system_status_and_performance/getting-started-with-tuned_monitoring-and-managing-system-status-and-performance[Getting started with TuneD] (RHEL documentation)
+* link:https://www.redhat.com/sysadmin/linux-tuned-tuning-profiles[How to manage tuning profiles in Linux] (Red Hat blog)
+
+//microshift-baseline is the name of the profile and used for automatic activation settings
+include::modules/microshift-low-latency-tuned-auto.adoc[leveloffset=+2]
+
+//additional resources for tuned profiles automatic activation
+[role="_additional-resources"]
+.Additional resources
+* xref:../../microshift_install/microshift-greenboot.adoc#greenboot-directories-details_microshift-greenboot[Greenboot directories details]
+
+//RHEL real-time kernel
+include::modules/microshift-low-latency-kernelrt-conc.adoc[leveloffset=+1]
+
+include::modules/microshift-low-latency-install-kernelrt-rpm.adoc[leveloffset=+2]
+
+include::modules/microshift-low-latency-install-kernelrt-rhel-edge.adoc[leveloffset=+2]
+
+[id="microshift-low-latency-install-kernelrt-rhel-edge-buildimage_{context}"]
+== Building the {op-system-ostree-first} image with the real-time kernel
+
+Start with the following procedure to complete the build process. Then complete the remaining steps in the installation documentation for installing {microshift-short} in a {op-system-ostree} image:
```

Comment:
> Start with the following procedure to complete the build process. Then complete the remaining steps...

Is this typical wording in the MS docs? It seems a bit unusual for technical docs; almost a tutorial format. 
```suggestion
To complete the build process, you must creating the RHEL for Edge image, as shown in the following section. Then complete the remaining steps in the installation documentation for installing {microshift-short} in a {op-system-ostree} image.
```

Reply:
Yes, our users have given us feedback that the repeated content is noisy for them. They'd rather have a link if it's in the same docs.